modules/loader.js

```diff
@@ -995,7 +995,7 @@ Loader.prototype.findGoodPeers = function(peers) {
 	peers = peers.filter(
 		item =>
 			// Remove unreachable peers or heights below last block height
-			item != null && item.height >= lastBlockHeight
+			item !== null && item.height >= lastBlockHeight
```

Comment:
That's incorrect - the previous check was checking the item against both of `null` and `undefined` values as explained in https://github.com/LiskHQ/lisk/pull/2562/files#r235950284, so that `item.height` doesn't result with `Uncaught ReferenceError: item is not defined`. 
In this case, we can replace this check with:
`item && item.height >= lastBlockHeight`, as we know that the height of the block cannot be equal to 0.